ZukoAzula2016/src/org/team1540/zukoazula/Shooter.java

```diff
@@ -13,18 +15,18 @@
 import ccre.frc.FRC;
 import ccre.time.Time;
 
-public class ShooterSpeedControl {
-    private static final ExtendedMotor shooterCAN = FRC.talonCAN(7);
-    private static BooleanInput shooting;
-    private static FloatOutput shooterMotor;
-    private static FloatInput gearTooth;
+public class Shooter {
+    private static final ExtendedMotor shooterLeftCAN = FRC.talonCAN(10);
+    private static final ExtendedMotor shooterRightCAN = FRC.talonCAN(11);
+    private static FloatInput encoder = FRC.encoder(0, 1, false, FRC.startTele);
+
+    private static ArbitratedBoolean shooterTrigger = ZukoAzula.behaviors.addBoolean();
 
     public static void setup() throws ExtendedMotorFailureException {
-        shooting = ZukoAzula.controlBinding.addBoolean("Shoot");
-        shooterMotor = shooterCAN.simpleControl();
-        gearTooth = FRC.counter(0, 1, FRC.startTele);
-        bangBangControl(gearTooth, shooting, ZukoAzula.mainTuning.getFloat("Shooter Velocity Target", 1), 0, 1).send(shooterMotor);
-        Cluck.publish("Shooter Motor", shooterMotor);
+        shooterTrigger.attach(ZukoAzula.teleop, ZukoAzula.controlBinding.addBoolean("Shoot"));
```

Comment:
Well, the arbitrator is only told to do anything in Teleop, and otherwise it'll default to false or zero. I think you'd need `shooterTrigger.attach(ZukoAzula.pit, ...)` as well?